openshift/templates/images/sidecar/docker/Dockerfile

```diff
@@ -40,18 +40,18 @@ RUN set -ex; \
 	# Fix up dpkg to run in a baby-root environment. (Needed for some apt packages).
 	mv /usr/bin/dpkg /usr/bin/ddpkg ; \
 	echo '#!/bin/bash\n/usr/bin/ddpkg --force-not-root "$@"\n' > /usr/bin/dpkg ; \
-	chmod +x /usr/bin/dpkg ; \
+	chmod +x /usr/bin/dpkg ;
+RUN set -ex; \	
 	# Install WP-CLI
-	curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar ; \
+	curl -L -o ./wp-cli.phar  https://github.com/wp-cli/wp-cli/releases/download/v2.12.0/wp-cli-2.12.0.phar ; \
```

Comment:
It might be worth thinking about using the SHA or the .asc file (included in the release) to validate the download when we do things like this.